### PR TITLE
Tentative fix for flaky specs in `backend/spec/features/admin/users_spec.rb`

### PR DESCRIPTION
**Pull request by spaghetticode**
**Description**

These specs are failing often on circleCI builds.
    
The added expectation ensures that the click toggled the table header HTML.
This has the double benefit to ensure HTML is consistent and leaves more time
to successfully verify the flaky expectation:

`expect(text_match_1).to appear_before text_match_2`

**Checklist:**

- [x] [Pull Request guidelines](https://github.com/solidusio/solidus/blob/master/CONTRIBUTING.md#pull-request-guidelines) are respected
- [ ] Documentation/Readme have been updated accordingly
- [x] Changes are covered by tests (if possible)
- [x] Each commit has a meaningful message attached that described WHAT changed, and WHY

### CODE REVIEW

File: backend/spec/features/admin/users_spec.rb [hunk 1]
```diff
@@ -57,19 +57,20 @@
   shared_examples_for 'a sortable attribute' do
     before { click_link sort_link }
 
-    it "can sort asc" do
+    it "can sort asc", :js do
```
Comment (kennyadsl) [inline backend/spec/features/admin/users_spec.rb]: what about this `, :js`? Why is this needed?

Comment (spaghetticode) [reply backend/spec/features/admin/users_spec.rb]: Not sure why it happens, but when using  `:rack_test` driver `expect(page).to have_selector '.sort_link.desc'` fails. It always finds `sort_link.asc` only.

These classes don't seem to be changed by javascript, as clicking the link generates a page reload, so I can't explain why:

```ruby
Capybara.current_driver = :selenium_chrome
=> :selenium_chrome
visit '/admin/users?q[s]=created_at+desc'
=> nil
expect(page).to have_selector '.sort_link.desc'
=> true
Capybara.current_driver = :rack_test
=> :rack_test
visit '/admin/users?q[s]=created_at+desc'
=> nil
expect(page).to have_selector '.sort_link.desc'
RSpec::Expectations::ExpectationNotMetError: expected to find visible css ".sort_link.desc" within #<Capybara::Node::Element tag="table" path="/html/body/div[5]/div/div/table"> but there were no matches
```
